dogen/schema/kwalify_schema.yaml

```diff
@@ -23,22 +23,14 @@ map:
           value: {type: str, required: True}
           description: {type: str}
   envs:
-    type: map
-    mapping:
-      information:
-        type: seq
-        sequence:
-          - type: map
-            mapping:
-              name: {type: str, required: True}
-              value: {type: any, required: True}
-              description: {type: str}
-      configuration:
-        seq:
-          - map:
-              name: {type: str, required: True}
-              example: {type: any}
-              description: {type: str}
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          name: {type: str, required: True}
+          value: {type: any}
```

Comment:
Correct, but why we should force people to quote anything? Besides this we loose information that this specific value is an integer.